src/Swashbuckle.AspNetCore.SwaggerGen/XmlComments/XmlCommentsDocumentHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Xml.XPath;
+
+namespace Swashbuckle.AspNetCore.SwaggerGen
+{
+    internal static class XmlCommentsDocumentHelper
+    {
+        internal static IReadOnlyDictionary<string, XPathNavigator> GetMemberDictionary(XPathDocument xmlDoc)
```

Comment:
If we're going for performance, this could return the concrete dictionary type to avoid the virtual dispatch over the interface as it's internal and no one _should_ be mutating it.